### ISSUE TITLE
VPN section: saying Bitcoin is an anonymous payment option

### ISSUE THREAD

**Issue by userdaphi** [privacyguides/privacyguides.org#553]
## the site is saying that Bitcoin (BTC) is an anonymous payment option. But this is not true. The ONLY privacyfriendly cryptocurrency is Monero (XMR). It is the only cryptocurrency which is private-by-design.

I give you an example:
IVPN
> Accepts Bitcoin
In addition to accepting credit/debit cards and PayPal, IVPN accepts Bitcoin, Monero and cash/local currency (on annual plans) as anonymous forms of payment.

People might think that Bitcoin is a private payment option. This is not true.

To include Bitcoin as a requirement for a private VPN provider is simply nonsense. It would be better to include Monero.
I would also suggest an explanation for that fact, why Bitcoin or any other cryptocoin is not private at all (except Monero of course)
**https://www.privacyguides.org/providers/vpn/**

**Comment by m77e4t** [comment]
Claiming 'only' Monero might be too much. There are other privacy coins like XMR which work similar to it or use other cryptographic methods to make them equal or more anonymous/private than XMR. https://www.coingecko.com/en/categories/privacy-coins

Still, the info regarding payment on VPN page as of 2/3 Feb 2022 seems pretty good.

**Comment by samuel-lucas6** [comment]
> There are other privacy coins like XMR 

Monero is XMR. The only other privacy coin I've heard of is Zcash, not that I'm following cryptocurrencies. If I remember correctly, some Zcash designers made some claims about potentially helping the police, which made people scared that it had a backdoor.

**Comment by m77e4t** [comment]
My bad. I made a mistake while writing the comment. I wanted to list a few privacy coins, but I accidentally wrote 'XMR' where it should not have been written (possibly I was doing some other stuff while writing the comment). Furthermore, I have corrected and updated my comment.

**Comment by dngray** [comment]
> People might think that Bitcoin is a private payment option. This is not true.

I've been thinking about this one. Now that we've launched the new site, I wouldn't mind a page that discusses cryptocurrencies in regard to their application of privacy (and not).

We've discussed it briefly within the team rooms. There is quite a lot of misconception about Bitcoin and how it relates to privacy, particularly the fact that it **is not private** and there is a **public ledger**.

It'd be an entry-level article, that discusses these aspects.
